src/components/common/Button/styles.js

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+
+const styles = (width, height, color, alignSelf, textColor, fontSize, letterSpacing, marginBottom, marginTop) =>
+  StyleSheet.create({
```

Comment:
This is something @peter7z corrected me, if you have dynamic styles based on parameters don't use StyleSheet.create, this is because this method creates a unique persisted object in memory every time the component is rendered, you can ask @peter7z for more information. So instead of returning StyleSheet.create..., just return an object, it will be more efficient.
```suggestion
  {
```